crates/ark/src/variables/variable.rs

```diff
@@ -1069,6 +1069,13 @@ impl PositronVariable {
         // Concrete nodes are objects that are treated as is. Accessing an element from them
         // might result in special node types.
 
+        // Check for the special access key `.Last.value`; we can only get the
+        // value for this object by evaluating it in R
+        if *access_key == String::from(".Last.value") {
+            let last_robj = harp::parse_eval_global(".Last.value")?;
```

Comment:
Same note here for last-value.

oh it might be worth creating `harp::last_value()`? We could use it in this file and in `interface.rs`.

Reply:
Done!